scripts/feed.js

```diff
@@ -27,127 +26,179 @@ function Feed(feed_urls)
-  }
+  };
 
-  this.update = async function()
-  {
+  this.update = async function() {
     this.get_entries();
-    setTimeout(function(){ r.portal.port_list_el.innerHTML = r.feed.get_feed_html(); }, 1000);
-  }
+    setTimeout(function() {
+      r.portal.port_list_el.innerHTML = r.feed.get_feed_html();
+    }, 1000);
+  };
 
-  this.get_entries = function()
-  {
+  this.get_entries = function() {
     var entries = [];
     var online_ports_count = 0;
 
-    var archive_promises = this.archives.map((archive) => (
+    var archive_promises = this.archives.map(archive =>
       this.get_feed(archive)
-        .then((feed_entries) => {
+        .then(feed_entries => {
           online_ports_count += 1;
           entries = entries.concat(feed_entries);
           this.debounced_sort_refresh(entries);
         })
-        .catch((e) => {
-          console.warn(`Unable to fetch, this feed appears to be offline: ${archive.url}`);
+        .catch(e => {
+          console.warn(
+            `Unable to fetch, this feed appears to be offline: ${archive.url}`
+          );
         })
-    ));
+    );
 
     Promise.all(archive_promises).then(() => {
-      console.log("DONE")
+      console.log("DONE");
       // Finished attempting to load all ports, maybe do something here?
     });
-  }
+  };
 
-  this.get_feed_html = function()
-  {
+  this.get_feed_html = function() {
     var feed_html = "";
-    for(name in r.feed.portals){
+    for (name in r.feed.portals) {
       var portal = r.feed.portals[name];
       var url = r.feed.urls[name];
-      var last_entry = portal.feed[portal.feed.length-1];
-      var is_active = last_entry ? Math.floor((new Date() - last_entry.timestamp) / 1000) : 999999;
-      var rune = portal.port.indexOf(r.portal.data.dat) > -1 || portal.dat === r.portal.data.dat ? "@" : "~";
-
-      if(!last_entry){ continue; }
-      if(is_active > 190000 && portal.name != r.portal.data.name){
-        feed_html += "<ln title='"+(timeSince(last_entry.timestamp))+"' class='dead' data-operation='un"+url+"'>"+rune+""+portal.name+"</ln>";
+      var last_entry = portal.feed[portal.feed.length - 1];
+      var is_active = last_entry
+        ? Math.floor((new Date() - last_entry.timestamp) / 1000)
+        : 999999;
+      var rune =
+        portal.port.indexOf(r.portal.data.dat) > -1 ||
+        portal.dat === r.portal.data.dat
+          ? "@"
+          : "~";
+
+      if (!last_entry) {
+        continue;
       }
-      else{
-        feed_html+= "<ln title='"+(timeSince(last_entry.timestamp))+"' class='"+(is_active < 150000 ? "active" : "inactive")+"'><a href='"+url+"'>"+rune+""+portal.name+"</a></ln>";
+      if (is_active > 190000 && portal.name != r.portal.data.name) {
+        feed_html +=
+          "<ln title='" +
+          timeSince(last_entry.timestamp) +
+          "' class='dead' data-operation='un" +
+          url +
+          "'>" +
+          rune +
+          "" +
+          portal.name +
+          "</ln>";
```

Comment:
Is there a way to configure prettier not to do this?  To me, at least, laying out the string like this makes it less readable.

Reply:
We can do `// prettier-ignore` on a per line basis, however I think if we use template strings it'll come out nicer, as we can decide when to break the string, e.g.

```js
feed_html = (
  `<ln title="${timeSince(last_entry.timestamp})" class="dead" data-operation="un${url}">` +
  rune + portal.name +
  "</ln>"
)
```

Or however.